push-sdk-swift/DeviceRegistration.swift

```diff
@@ -133,9 +137,18 @@ open class DeviceRegistration: NSObject, URLSessionTaskDelegate {
             }
             
             // Fail if not all config mandatory items are present
-            assert(clientInfoObject.deviceToken != nil, "'token' should be set")
-            assert(clientInfoObject.variantID != nil, "'variantID' should be set")
-            assert(clientInfoObject.variantSecret != nil, "'variantSecret' should be set");
+            if clientInfoObject.deviceToken == nil {
```

Comment:
I think a `guard` would be more elegant sth like:
    guard let deviceToken = clientInfoObject.deviceToken else {
        failure(...)
        return
    }
then outside the guard you could rewrite line 154
 UserDefaults.standard.set(deviceToken, forKey: "deviceToken")
Wdyt?